content/docs/gke/private-clusters.md

```diff
@@ -7,65 +7,208 @@ weight = 5
 This guide describes how to create private clusters with Kubeflow on Google 
 Kubernetes Engine (GKE).
 
+Creating a [private Kubernetes Engine cluster](https://cloud.google.com/kubernetes-engine/docs/how-to/private-clusters)
+means the Kubernetes Engine nodes won't have public IP addresses. This can improve security by blocking unwanted outbound/inbound
+access to nodes. Removing IP addresses means external services (including GitHub, PyPi, DockerHub etc...) won't be accessible
+from the nodes. Google services (BigQuery, Cloud Storage, etc...) are still accessible.
+
+Importantly this means you can continue to use your [Google Container Registry (GCR)](https://cloud.google.com/container-registry/docs/) to host your Docker images. Other Docker registries (for example DockerHub) will not be accessible. If you need to use Docker images
+hosted outside Google Container Registry you can use the scripts provided by Kubeflow
+to mirror them to your GCR registry.
+
 ## Before you start
 
-This guide assumes you have already set up Kubeflow with GKE. If you haven't done
-so, follow the guide to [deploying Kubeflow on GCP](/docs/gke/deploy/).
+Before installing Kubeflow ensure you have installed the following tools:
+    
+  * [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+  * [gcloud](https://cloud.google.com/sdk/)
+  * [jsonnet](https://github.com/google/jsonnet/releases)
```

Comment:
is this optional only if one need to mirror additional images?
if so maybe better call it out here.

Reply:
Mirroring images isn't really optional. Some of the critical images in Kubeflow need to be mirrored.